README.md

```diff
@@ -31,31 +30,36 @@ Make sure you have at least 1.5 GB free disk space.
     sudo apt install libssl-dev libpq-dev libc6-dev curl gcc pkg-config
     ```
 
-3.  *ff-node-monitor* is written in [Rust](https://www.rust-lang.org/) using
+3.  Fetch the sources:
+    
+    ```
+    sudo -u ff-node-monitor git clone https://github.com/freifunk-saar/ff-node-monitor.git /opt/ff-node-monitor/src
```

Comment:
Please use `~ff-node-monitor` instead of `/opt/ff-node-monitor`.

Reply:
I think, we had this earlyer, and decided against the notation `~ff-node-monitor`, Maybe it was not working on all systems like this?

Reply:
You accepted the change in #46. I personally don't think the home-dir writing is so clear, you have to think around the corner, where the user has its home dir and you only save 4 characters anyway

Reply:
We are using `~ff-node-monitor` in some places though, and IMO it is clearer than `/opt`.

But I think it is easier if I just edit the README the way I like it, rather than going back and forth here.^^ I'll do that once I find some time.